package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "test": "mocha --require ignore-styles,@babel/register",
     "test:watch": "mocha --require ignore-styles,@babel/register -w",
+    "test:fix-lint": "eslint test --fix",
```

Comment:
I don't think this task should be called "test"... maybe just "lint" or "eslint"

Reply:
This is the linter for the folder test, and it has it’s owned eslint config, do you think we also need to apply no semi in the test spec?

Note that client code and test code use different eslint config

Reply:
Oh, I didn't notice that. Why do they have different configs?

Reply:
There is config for client code, 

and one for mocha, i think that’s the reason mocha has the variable describe and it, and runs with node script

https://github.com/wikimedia/eslint-config-wikimedia